tests/platform_test.cpp

```diff
@@ -60,6 +57,24 @@ BOOST_FIXTURE_TEST_CASE(add_casino, platform_tester) try {
 
 } FC_LOG_AND_RETHROW()
 
+BOOST_FIXTURE_TEST_CASE(vesion_test, platform_tester) try {
+    account_name casino_account = N(casino.1);
+
+    create_account(casino_account);
+
+    base_tester::push_action(platform_name, N(addcas), platform_name, mvo()
```

Comment:
why do you call `addcas` action

Reply:
Version initialization located at contract class constructor, so I have to call some action to initialize version table